tests/testthat/helper-data.R

```diff
@@ -40,3 +40,13 @@ h_get_data_mixture <- function() {
     nObsshare = 4L
   )
 }
+
+h_get_data_DA <- function() { # nolintr
```

Comment:
```suggestion
h_get_data_augmented <- function() {
```
and similarly in the other test file where it is used